src/main/java/xyz/gvital/app/UserNotFoundException.java

```diff
@@ -0,0 +1,6 @@
+package xyz.gvital.app;
+
+public class UserNotFoundException extends RuntimeException {
+    public UserNotFoundException(Long userId) {
+    }
+}
```

Comment:
Optional changes: This is a good practice to provide a meaningful message for more context when the exception is thrown.

```suggestion
public class UserNotFoundException extends RuntimeException {
    public UserNotFoundException(Long userId) {
        super("User with ID " + userId + " not found.");
    }
}
```